llvm/lib/ExecutionEngine/JITLink/EHFrameSupport.cpp

```diff
@@ -18,40 +18,40 @@
 namespace llvm {
 namespace jitlink {
 
-EHFrameSplitter::EHFrameSplitter(StringRef EHFrameSectionName)
-    : EHFrameSectionName(EHFrameSectionName) {}
+SectionSplitter::SectionSplitter(StringRef SectionName)
```

Comment:
Should we call this DwarfSectionSplitter? It can split any sections that use the DWARF format of starting each contribution with a length field.